src/repos.cc

```diff
@@ -1883,6 +1911,11 @@ void add_repo_from_file( Zypper & zypper,
     if ( !indeterminate(gpgCheck) )
       repo.setGpgCheck(gpgCheck);
 
+    long long prio = parse_priority(zypper);
```

Comment:
like in add_repo_by_url; ```unsigned ...``` and the call should be moved outside the loop; best before ```list<RepoInfo> repos;```.